### DISCUSSION TITLE
Azure Marketplace: Enable offer for subscriptions of Microsoft Azure Cloud Solution Providers

### DISCUSSION THREAD

**Discussion by Joly0** [nextcloud/all-in-one#5483] [Ideas] [answered]
### Steps to reproduce
1. Go to Azure Marketplace
2. Try to install AIO
3. See error

### Expected behavior 
Dont see error

So i am trying to install Nextcloud AIO for my company as a replacement for a really old nextcloud installation. I thought AIo might be a good fit, as its easy to maintain and update. I found out, that there is AIO avaialble through azure marketplace, but i can install it.
Under my subscription i only get the message

> This offer is not available for subscriptions from Microsoft Azure Cloud Solution Providers

It looks like this could be the solution:
https://learn.microsoft.com/en-us/answers/questions/623991/this-offer-is-not-available-for-subscriptions-from

Or is this intended not to work for csp subscriptions?

**Comment by szaimen** [comment, marked as answer]
Hi, thanks for your report.

> Or is this intended not to work for csp subscriptions?

No, of course it should work for everyone. (we did not intentionally do this)

But I fear we do not have the required knowledge and resources to fix this.

However as a workaround, you could spin up your own VM on Azure for example based on Ubuntu server 24.04 LTS, install docker and AIO afterwards. See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-use-this

**Reply by Joly0** [reply]
Thanks for the information.
As far as i can see, this is "easily" configurable in the Microsoft Partner Center https://learn.microsoft.com/en-us/partner-center/marketplace-offers/cloud-solution-providers#how-to-configure-an-offer 
Is there a place where i could request this change?
Also something that i noticed here is, that the VM thats being generated by your marketplace offer is a "Gen 1" VM as far as i can see. It would be great, if this was configurable or if it was a "Gen 2" VM https://learn.microsoft.com/en-us/partner-center/marketplace-offers/azure-vm-plan-technical-configuration#image-types
